solidity/dashboard/src/connectors/utils.js

```diff
@@ -24,7 +24,10 @@ export const getChainIdFromV = (vInHex) => {
 }
 
 export const getChainId = () => {
-  if (process.env.NODE_ENV === "development") {
+  if (
+    process.env.NODE_ENV === "development" ||
+    process.env.NODE_ENV === "test"
+  ) {
```

Comment:
I know it's pretty small change, but maybe it would be a good idea to make such changes in a separate PR since it's not really related to TBTC v2? 🤔 

Don't need to change it, it's just a thought and I'm curious what you think.

Reply:
I decided to push this change because of failing tests. If we use a different chain ID other than that specified in [`config.json`](https://github.com/keep-network/keep-core/blob/main/solidity/dashboard/src/config/config.json) file the tests fail. So w/o this change, I was not able to check if all tests passed.